tests/unit/phpDocumentor/DocumentationFactoryTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.5
+ *
+ * @copyright 2010-2015 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor;
+
+use Mockery as m;
+use phpDocumentor\Project\Version\Definition;
+use phpDocumentor\Project\VersionNumber;
+
+/**
+ * Class DocumentationFactoryTest
+ * @coversDefaultClass phpDocumentor\DocumentationFactory
+ */
+class DocumentationFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var DocumentationFactory
+     */
+    private $fixture;
+
+    /**
+     * @var m\Mock
+     */
+    private $documentGroupFactoryMock;
+
+    protected function setUp()
+    {
+        $this->fixture = new DocumentationFactory();
+        $this->documentGroupFactoryMock = m::mock(DocumentGroupFactory::class);
+    }
+
+    /**
+     * @covers ::create
+     */
+    public function testMinimalDocumentationCreation()
+    {
+        $versionDefinition = new Definition(new VersionNumber('1.0.0'));
+
+        $documentation = $this->fixture->create($versionDefinition);
+
+        $this->assertEquals(new VersionNumber('1.0.0'), $documentation->getVersionNumber());
+    }
+
+    /**
+     * @covers ::create
+     * @covers ::<private>
```

Comment:
I did not know about these constructs; thanks! I will apply them as well in my tests